test/fixtures/version-6/can-observable-object/observable-object-static-properties-input.js

```diff
@@ -0,0 +1,11 @@
+const Todo = DefineMap.extend('Foo', { bar: 'Bar', seal: false}, {
```

Comment:
I think we should just handle `seal: true` correctly and ignore any other static properties here.